docs/verify/local-testing-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Qiskit Runtime local testing mode
+description: Use local testing mode to simulate quantum hardware in Qiskit Runtime
+---
+
+<span id="test-locally"></span>
+# Qiskit Runtime local testing mode
+
+Quantum simulators should be used to help develop and test programs before fine-tuning them and sending them to real quantum hardware. Local simulators can do this with good performance and efficiency.  Clifford circuits can be simulated very efficiently, and results can be verified. This is useful to give confidence in an experiment.
+
+To use local testing mode with Qiskit Runtime, specify one of the fake backends in ``qiskit_ibm_runtime.fake_provider`` or specify a Qiskit Aer backend when instantiating a primitive or a session.  For details, see the [fake_provider API reference.](/api/qiskit-ibm-runtime/fake_provider)
+
+If you currently use Qiskit (and not Qiskit Runtime), you can use the existing Qiskit Aer primitives for local testing.  See [exact and noisy simulation with Qiskit Aer primitives](../verify/simulate-with-qiskit-aer) for details. 
```

Comment:
instead of saying "and not qiskit runtime" I think you can just say "if you currently use the qiskit SDK". you can still use the qiskit primitives and qiskit aer primitives even if later you're going to user runtime to send the job, but the way this sentence is currently phrased implies that you should only use them if you're not also planning to use runtime